app/src/main/java/com/infomaniak/drive/data/cache/HandleSchemaVersionBelowZero.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Infomaniak kDrive - Android
+ * Copyright (C) 2025 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.drive.data.cache
+
+import io.realm.DynamicRealm
+import io.realm.Realm
+import io.realm.RealmConfiguration
+import io.realm.RealmMigration
+
+object HandleSchemaVersionBelowZero {
+    private val throwOldSchemaVersionRealmConfiguration = DriveInfosController.baseDriveInfosRealmConfigurationBuilder
+        .migration(DetectTooOldSchemaMigration())
+        .build()
+
+    private val deleteTooOldSchemaVersionRealmConfiguration = DriveInfosController.baseDriveInfosRealmConfigurationBuilder
```

Comment:
I think we can simplify this name, as it's really just a `deleteDataMigration`or `dropDataMigration`

Reply:
I took your idea and went with deleteRealmConfiguration to keep the same naming pattern